src/api/tests/user/verify-email.test.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from 'chai'
+import app from '../../app'
+import supertest from 'supertest'
+
+import sakiewkaCrypto from 'sakiewka-crypto'
+const { constants, user } = sakiewkaCrypto
+
+// @ts-ignore
+const mockFn = jest.fn(() => {
+  return new Promise((resolve: Function) => {
+    resolve('test wallet')
+  })
+})
+
+user.verifyEmail = mockFn
+
+describe('/btc/user/verify-email', () => {
+  it('should exist', async () => {
+    const response = await supertest(app)
+      .get(`/${constants.BASE_API_PATH}/user/verify-email`)
+
+    expect(response.status).to.not.equal(404)
+  })
+
+  it('should not accept request with missing query params', async () => {
+    const response = await supertest(app)
+      .get(`/${constants.BASE_API_PATH}/user/verify-email`)
+
+    expect(response.status).to.be.equal(400)
+    expect(response.body.error.message).to.be.equal('"code" is required')
```

Comment:
Hmm, a czemu nie `email` jest required?

Reply:
Odpowiedź powinna zawierać albo wszystko brakujące paramsy, albo zrób test jak poniżej zaproponowałem, gdzie explicite brakuje zawsze tylko jednego paramsa

Reply:
Bo weryfikacja odbywa sie w pewnej kolejnosci i zawsze najpierw jest code a pozniej emial.

Reply:
Jak zmieni sie kolejnosc, to zmieni sie odpowiedz. Just sayin ;) 

(Zostaw jak jest, ale moze to prowadzic w przyszlosci do glupio failujacych testow)

Reply:
Znowu, konwencja ale zgadzam sie.